src/app/external-cluster-wizard/steps/external-cluster/provider/gke/component.ts

```diff
@@ -280,17 +346,19 @@ export class GKEClusterSettingsComponent
     }
   }
 
-  private _getGKEMachineSizesForMachineDeployment(): void {
+  private _getGKEMachineTypesForMachineDeployment(): void {
+    this.machineTypeLabel = MachineTypeState.Loading;
     this._externalMachineDeploymentService
-      .getGKEMachineSizesForMachineDeployment(this.projectID, this.cluster.id)
-      .pipe(
-        map((machineSizes: GCPMachineSize[]) =>
-          machineSizes.map((machineType: GCPMachineSize) => {
-            return `${machineType.name} ${machineType.description})`;
-          })
-        )
-      )
-      .subscribe((machineTypes: string[]) => (this.machineTypes = machineTypes));
+      .getGKEMachineTypesForMachineDeployment(this.projectID, this.cluster.id)
+      .subscribe((machineTypes: GCPMachineSize[]) => {
+        machineTypes.map(machineType => {
```

Comment:
Check this comment: https://github.com/kubermatic/dashboard/pull/4917#discussion_r960396002